File: Python/somaImpares.py
x: int
y: int
troca: int
soma: int

print("Digite dois números:")
x = int(input())
y = int(input())

if x > y:
    troca = x
    x = y
    y = troca

soma = 0
for i in range(x+1, y):
    if i % 2 != 0:
        soma += i

print("SOMA =", soma)

File: Cpp/soma_impares.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{

    int x, y, i, soma, troca;

    cout << "Digite dois numeros:" << endl;
    cin >> x >> y;

    if(x > y)
    {
        troca = x;
        x = y;
        y = troca;
    }

     soma = 0;

    for(i = x+1; i < y; i++)
        {
            if (i%2 != 0)
            {
                soma = i + soma;
            }
        }

    cout << "SOMA DOS IMPARES = " << soma;

    return 0;
}
